packages/flame/CHANGELOG.md

```diff
@@ -2,27 +2,28 @@
 
 ## [Next]
  - Fix link to code in example stories
- - Fix RotateEffect with negative deltas
- - Add isDragged to Draggable
- - Fix anchor of rendered text in TextComponent
- - Add new extensions to handle math.Rectangles nicely
+ - Fix `RotateEffect` with negative deltas
+ - Add `isDragged` to `Draggable`
+ - Fix the anchor of rendered text in `TextComponent`
+ - Add new extensions to handle `math.Rectangles` nicely
  - Implement color parsing methods
  - Migrated the `Particle` API to `Vector2`
- - Add copyWith function to TextRenderer
- - Fix debug mode is not propagated to children of non-Position components
- - Fix size property of TextComponent was not correctly set
- - Fix anchor property was being incorrectly passed along to text renderer
+ - Add `copyWith` function to `TextRenderer`
+ - Fix debug mode is not propagated to children of non-`Position` components
+ - The `size` property of `TextComponent` was not correctly set
+ - The `anchor` property was being incorrectly passed along to `TextRenderer`
  - All components take priority as an argument on their constructors
- - Fix renderRotated
- - Use QueryableOrderedSet for Collidables
- - Refactor TextBoxComponent
- - Fix bugs with TextBoxComponent
+ - Fix `renderRotated`
+ - Use `QueryableOrderedSet` for `Collidable`s
+ - Refactored `TextBoxComponent`
```

Comment:
Refactor

Reply:
The `Refactored` was based on the other entries in our CHANGELOG where it started the sentence with it.

Reply:
We shouldn't change the lines in the changelog after they have been merged, then you lose the ability to directly see in which PR which entry is related to. I think that is more important than having the backticks there.

Reply:
Indeed, how much it will be annoying for me to see, I should not pollute the reference to the original PR. Will revert these changes.